### DISCUSSION TITLE
Super Slicer & MSLA Support??

### DISCUSSION THREAD

**Discussion by BuckyTheElk** [supermerill/SuperSlicer#2805] [General]
Will Super Slicer ever expand and offer support for MSLA printers like the Anycubic Photon M3 Max or other brands?

**Comment by supermerill** [comment]
Maybe. Not a priority.
The slicing & path planning is easy, as it can overlap without any bad consequence.
If the output is the standard gcode, output is easy.
The gui is a bit less easy, as "printer type" is currently spagetty code. It can be circumvented for testing though.
